src/math/sieve_of_eratosthenes.jl

```diff
@@ -0,0 +1,18 @@
+"""
+Sieve of Eratosthenes is an algorithm for finding all the primes upto a limit `n`.
+Reference:
+-https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes
+
+Contributed By:- [Ved Mahajan](https://github.com/Ved-Mahajan)
+"""
+function eratosthenes(n)
```

Comment:
This function is missing documentation. Please add documentation to the function.